subsys/bluetooth/host/hci_core.c

```diff
@@ -4683,16 +4683,14 @@ static void le_per_adv_sync_lost(struct net_buf *buf)
 	term_info.addr = &per_adv_sync->addr;
 	term_info.sid = per_adv_sync->sid;
 
-	/* Clearing bit before callback, so the caller will be able to restart
+	/* Deleting before callback, so the caller will be able to restart
 	 * sync in the callback
 	 */
-	atomic_clear_bit(per_adv_sync->flags, BT_PER_ADV_SYNC_SYNCED);
```

Comment:
I am assuming the fail is clear elsewhere now?

Reply:
`per_adv_sync_delete` (which replaces this lane), clears all flags